docs/organisation-actions.md

```diff
@@ -0,0 +1,14 @@
+# Organisation actions
+
+## Listing
+Lists all of the current organizations. You can do this with `orgs list`:
```

Comment:
Probably emphasise organizations that can be accessed by that token. It's not all organizations.